body_to_camera: {
  qx: 0,
  qy: 0,
  qz: 0,
  qw: 1.0,
  tx: 0,
  ty: 0,
  tz: 0}
